### ISSUE TITLE
Support Query parameters

### ISSUE THREAD

**Issue by rushton** [prestosql/presto-python-client#21]
Currently query parameters are ignored: https://github.com/prestosql/presto-python-client/blob/master/presto/dbapi.py#L234-L238

I imagine the functionality would be something similar to pyhive:
```python
cursor.execute("SELECT * FROM foo WHERE x = %s AND y = %s", ["foo", 1])
```

resulting in a query of 
```SQL
SELECT * FROM foo WHERE x = 'foo' AND y = 1
```

At current, the above python code results in a query parse exception.

**Comment by findepi** [comment]
Ignoring arguments -- https://github.com/prestosql/presto-python-client/pull/22

let me reword this issue to actually add support for them

**Comment by rushton** [comment]
fixed by #41
